src/comments.js

```diff
@@ -919,8 +919,35 @@ function printComment(commentPath, options) {
     case "Comment":
       return "#" + comment.value.trimRight();
     case "CommentBlock":
-    case "Block":
-      return "/*" + comment.value + "*/";
+    case "Block": {
+      const lines = comment.value.split("\n");
+      const isJsDocComment =
+        lines.length > 1 &&
+        lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*");
+      if (!isJsDocComment) {
+        return "/*" + comment.value + "*/";
+      }
+
+      const tabWidth = options.tabWidth;
+      const indentSize = util.getIndentSize(comment.value, tabWidth);
+
+      const parts = [];
+
+      lines.forEach((line, index) => {
+        parts.push(
+          docBuilders.addAlignmentToDoc(
+            line.replace(/^[ \t]+/, " "),
+            indentSize,
+            tabWidth
+          )
+        );
+        if (index < lines.length - 1) {
+          parts.push(docBuilders.hardline);
+        }
+      });
+
+      return docBuilders.concat(["/*", ...parts, "*/"]);
```

Comment:
Node v4 does not support `...` operator, use `concat(parts)` instead.

Reply:
@ikatyang Good eye! Thank you :smile: